### PR TITLE
Add a deprecation warning for django 1.8 and 1.9

### PR DESCRIPTION
**Pull request by syldlb**
## Review

* [x] Tests
* [x] `CHANGELOG.rst` Updated
* [ ] Delete your branch

### CODE REVIEW

File: formidable/app.py [hunk 1]
```diff
@@ -21,3 +23,11 @@ def ready(self):
         """
         from .views import check_callback_configuration
         check_callback_configuration()
+
+        # Deprecation warning for django 1.8 and 1.9
+        if django.VERSION[0] is 1:
```
Comment (brunobord) [inline formidable/app.py]: you'd better not use this... Please use version comparison tools, like: https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python

File: formidable/app.py [hunk 2]
```diff
@@ -21,3 +23,11 @@ def ready(self):
...
+            minorVersion = django.VERSION[1]
```
Comment (brunobord) [inline formidable/app.py]: Please don't use camelCase variable names.

File: formidable/app.py [hunk 3]
```diff
@@ -21,3 +23,11 @@ def ready(self):
...
+            if minorVersion is 8 or minorVersion is 9:
+                warnings.warn("Support for django 1. " + str(minorVersion) +
```
Comment (brunobord) [inline formidable/app.py]: I wouldn't do a dynamic message here.
I'd write:

> Support for Django 1.8 and 1.9 is about to be dropped. Please update to a newer version.

Comment (syldlb) [reply formidable/app.py]: I've updated the code according to your feedbacks!

File: formidable/app.py [hunk 4]
```diff
@@ -21,3 +25,9 @@ def ready(self):
         """
         from .views import check_callback_configuration
         check_callback_configuration()
+
+        # Deprecation warning for django 1.8 and 1.9
+        if StrictVersion(django.get_version()) <= StrictVersion("1.9"):
```
Comment (brunobord) [inline formidable/app.py]: here, you only warn for Django 1.8.

The correct line:

```python
if StrictVersion(django.get_version()) < StrictVersion("1.10"):
```

Tested using this:

```python
        # Deprecation warning for django 1.8 and 1.9
        if StrictVersion(django.get_version()) < StrictVersion("1.10"):
            raise Exception("MEUH")
            warnings.warn("Support for Django 1.8 and 1.9 is about to be "
                          "dropped. Please update to a newer version.",
                          DeprecationWarning)

```

If you run your former condition, you see that the tests fail only for `django18-*`.

Comment (syldlb) [reply formidable/app.py]: I'm running the demo site with Django 1.9 and I do have the warning raised, that's what tricked me. Indeed with Django 1.9.X it won't work, as I would have found launching tox with 1.9. I'm disappointed with myself :thinking:

**Comment by syldlb** [comment]
done ! :)

**Pull request merged**